parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py

```diff
@@ -203,42 +364,21 @@ def assert_task_args(
         else:
             conversations_needed = shared_state.conversations_needed
         args.blueprint.num_conversations = sum(conversations_needed.values())
-        super().assert_task_args(args, shared_state)
-        assert (
-            args.blueprint.get("task_description_file", None) is not None
-        ), "Must provide a task description file"
-        full_path = os.path.expanduser(args.blueprint.task_description_file)
-        assert os.path.exists(
-            full_path
-        ), f"Target task description path {full_path} doesn't exist"
+        super().assert_task_args(args=args, shared_state=shared_state)
 
-        assert (
-            args.blueprint.get("left_pane_text_path", None) is not None
-        ), "Must provide a left pane text file"
-        full_path = os.path.expanduser(args.blueprint.left_pane_text_path)
-        assert os.path.exists(
-            full_path
-        ), f"Target left pane text path {full_path} doesn't exist"
-
-        assert (
-            args.blueprint.get("annotations_config_path", None) is not None
-        ), "Must provide an annotation config file"
-        full_path = os.path.expanduser(args.blueprint.annotations_config_path)
-        assert os.path.exists(
-            full_path
-        ), f"Target annotation config path {full_path} doesn't exist"
-
-        assert (
-            args.blueprint.get("onboard_task_data_path", None) is not None
-        ), "Must provide an onboarding data file"
-        full_path = os.path.expanduser(args.blueprint.onboard_task_data_path)
-        assert os.path.exists(
-            full_path
-        ), f"Target onboarding data path {full_path} doesn't exist"
+        if args.blueprint.get("annotations_config_path", "") != "":
```

Comment:
Ah, thanks for pointing that out - we only care about an onboarding data file if we're doing annotations, since the onboarding involves learning how to do annotations. I've added a comment to explain that